src/pages/AboutUs/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react'
+import styles from './styles.module.css'
+import CardGithub from '../../components/Card/CardGithub';
+import CardDiscord from '../../components/Card/CardDiscord';
+import Contribuitors from '../../components/Contributors';
+import MobileCardDiscord from '../../components/Card/MobileCardDiscord'
+import MobileCardGithub from '../../components/Card/MobileCardGithub'
+import { useTranslation } from 'react-i18next';
+
+export default function AboutUs() {
+
+    const { t } = useTranslation();
+    
+    return (
+        <>
+        <section className={styles.aboutUs}>
+            <section className={styles.firstStepAboutUS}>
+                <div className={styles.firstStepContentWrapper}>
+                    <h2>{t("about.mainTitle")}</h2>
+                    <p>{ t('about.firstPartDescription')}</p>
+                    <p>
+                        { t('about.secondPartDescription')}
+                        <strong>{ t('about.openDevName')}</strong>
+                        { t('about.thirdPartDescription')}
+                    </p>
+                </div>
+                <div className={styles.firstStepGraphWrapper}></div>
+            </section>
+
+            <section className={ styles.secondStepAboutUs}>
+                <div className={ styles.secondStepContentWrapper}>
+                    <h2>{t("about.collaborationsTitle")}</h2>
+                    <p>
+                        {t("about.firstPartCollaborationsDescription")}
+                        <strong>{t("about.openSource")}</strong>
+                        {t("about.secondPartCollaborationsDescription")}
```

Comment:
Me parece muito confuso esse esquema de first, second, third Part. Agora com o uso do `<Trans />` pode usar o próprio nome da região.

```suggestion
            <section className={ styles.collaborations}>
                <div className={ styles.collaborationsWrapper}>
                    <h2>{t("about.collaborationsTitle")}</h2>
                    <p>
                        {t("about.collaborationsDescription")}
                    </p>
```